test/odbc/drivers.cpp

```diff
@@ -0,0 +1,93 @@
+#include "drivers.h"
+#include <fstream>
+
+using std::pair;
+
+Drivers::Drivers() {
+  SetDrivers();
```

Comment:
Rather than calling SetDrivers and parse the config file every time we initialize the constructor, can we make the `odbc_drivers_` member variable static and do the following?

```
if (odbc_drivers_.isEmpty())
   SetDrivers()
```

This way, we only need to parse through the file during the first time this object is created.

Reply:
Good suggestion. Done.